### DISCUSSION TITLE
Factors that influence radius and Elevation (Degrees)

### DISCUSSION THREAD

**Discussion by miguel-merlin** [stevensblueprint/nycmesh-node-visualizer#38] [General]
![Radiation-pattern-of-a-realistic-directional-antenna png](https://github.com/stevensblueprint/nycmesh-node-visualizer/assets/106508795/8d6fde0a-cde1-4bfb-91c2-d3ebbe2229b3)

## Average Radius of a Sector Lobe of an Antenna
The average radius of a sector lobe of an antenna depends on various factors such as the antenna design, frequency, and beamwidth. This README provides an overview of the key points to consider when determining the average radius of a sector lobe.

## Antenna Radiation Pattern
The radiation pattern of an antenna describes how the radiated power is distributed in different directions. It consists of main lobes and sidelobes. The main lobe is the region of the radiation pattern that contains the highest power or exhibits the greatest field strength. The average radius of the sector lobe can be determined by analyzing the beamwidth of the main lobe.

## Beamwidth
The beamwidth of an antenna is the angular width of the main lobe. It is usually specified by the half power beam width (HPBW), which is the angle encompassed between the points on the side of the lobe where the power has fallen to half (-3 dB) of its maximum value. The larger the beamwidth, the wider the sector lobe.

## Antenna Design Factors
The average radius of a sector lobe can also be influenced by the specific design of the antenna, such as the shape and size of the aperture or radiating elements. Larger antennas generally have narrower main lobes and narrower sidelobes. The design parameters and construction of the antenna can affect the shape and characteristics of the radiation pattern.

## Frequency and Bandwidth
The frequency of operation and bandwidth of the antenna can also impact the radiation pattern. Antennas transmit and receive signals with frequency components within a narrow band. The impedance and characteristics of the antenna may vary with frequency, which can affect the shape and size of the main lobe and sidelobes.

## Antenna Efficiency
Antenna efficiency, also known as radiation efficiency, describes the losses in an antenna due to resistive losses. Antenna efficiency can affect the overall performance and power radiated by the antenna. It is usually expressed as a percentage and can vary depending on the type of antenna.

**Comment by ezrizhu** [comment]
This doesn't have to be super accurate, we can just copy-and-paste the malefactor's numbers on lobe size.
